cpp/src/io/utilities/column_buffer.hpp

```diff
@@ -67,11 +67,6 @@ inline rmm::device_buffer create_data(
  * will be used to create a column.
  */
 struct column_buffer {
-  // there is a potential bug here.  In the decoding step, the buffer of
-  // data holding these pairs is cast to an nvstrdesc_s, which is a struct
-  // containing <const char *, size_t>.   So there is a mismatch between the
-  // size_type and the size_t.  I believe this works because the str_pair is
-  // aligned out to 8 bytes anyway.
   using str_pair = thrust::pair<const char*, size_type>;
```

Comment:
this alias is declared in a few places in libcudf, with name `string_index_pair`. I think it would be neat to be consistent with those aliases, so it's easier to see that it's the same type.
```suggestion
  using string_index_pair = thrust::pair<const char*, size_type>;
```